server/src/core/user/index.ts

```diff
@@ -1,2 +1,4 @@
 export { User, UserCreateArgs, UserService, UserRole } from './types';
-export { Users } from './user-service';
\ No newline at end of file
+export { Users } from './user-service';
+export * as Validator from './validator';
```

Comment:
I don't think we should export the validator outside the user modules, because it's only used internally in the user service. If we do need it outside this module, then at that point we can export it.